### ISSUE TITLE
Change 'member' to 'adherent' in htdocs/adherents/class/api_members.class.php

### ISSUE THREAD

**Issue by floweb** [EuskalMoneta/dolibarr#6]
In htdocs/adherents/class/api_members.class.php, line 202 should be:

```php
if( ! DolibarrApi::_checkAccessToResource('adherent',$member->id)) {
```
instead of:

```php
if( ! DolibarrApi::_checkAccessToResource('member',$member->id)) {
```

**Comment by floweb** [comment]
ping @Xebax

**Comment by Xebax** [comment]
Fixed in 4d46180086169e6dcec95cb6c8d134114ef72b4b.
